20240517-R-http-metrics-path-matching.md

```diff
@@ -75,15 +69,12 @@ dapr_http_server_request_count{app_id="ping",method="InvokeService/ping",status=
 http:
   increasedCardinality: false
   pathMatching:
-    ingress:
-    - /orders/{orderID}
-    egress:
     - /orders/{orderID}
 ```
 
 ```
 dapr_http_server_request_count{app_id="ping",method="GET",path="/orders/{orderID}",status="200"} 4
-dapr_http_server_request_count{app_id="ping",method="GET",path="_",status="200"} 1
+dapr_http_server_request_count{app_id="ping",method="GET",path="/",status="200"} 1
```

Comment:
Does "/" mean the request was to "/" or it was a path not matched?

Reply:
in this case unmatched

Reply:
changed to empty string, do you agree?

Reply:
unmatched path should not overlap with the metric of an actual call to "/". Thanks for changing this to be "". Empty string IMO better reflects what this actually is: path is not present.